### ISSUE TITLE
Install error 

### ISSUE THREAD

**Issue by EmilyHoedt** [adw96/DivNet#65]
Hello
I have installed breakaway, but when I try to install DivNet I get the following error.
A similar error occurred when I tried to install ampvis2 previously and fixed it by `devtools::install_github("tidyverse/ggplot2")`
Any suggestions for successful DivNet install would be greatly appreciated.
Thank you,
Emily

`> remotes::install_github("adw96/DivNet")
Downloading GitHub repo adw96/DivNet@HEAD
Skipping 1 packages not available: phyloseq
√  checking for file 'C:\Users\ech132\AppData\Local\Temp\1\RtmpWgjvz0\remotes617014cd2c52\adw96-DivNet-31e04e2/DESCRIPTION' (394ms)
-  preparing 'DivNet':
√  checking DESCRIPTION meta-information ... 
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'DivNet_0.3.6.tar.gz'
   
* installing *source* package 'DivNet' ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error: (converted from warning) package 'phyloseq' was built under R version 4.0.3
Execution halted
ERROR: lazy loading failed for package 'DivNet'
* removing 'C:/Program Files/R/R-4.0.2/library/DivNet'
Error: Failed to install 'DivNet' from GitHub:
  (converted from warning) installation of package ‘C:/Users/ech132/AppData/Local/Temp/1/RtmpWgjvz0/file61703f094f39/DivNet_0.3.6.tar.gz’ had non-zero exit status`

**Comment by adw96** [comment]
Hi @EmilyHoedt -- could you please provide your R version and session info? An example of how to do so is here: adw96/breakaway#79

I expect that the issue is that you have not yet installed `phyloseq`

Cheers,

Amy

**Comment by EmilyHoedt** [comment]
Hi Amy,
Thanks very much for your reply. I had installed phyloseq and breakaway, but still ran into problems with DivNet. I've added the details for my R version and session below.
Thanks again


> library(breakaway)
> library(phyloseq)
Warning message:
package ‘phyloseq’ was built under R version 4.0.3 
> library(magrittr)
Warning message:
package ‘magrittr’ was built under R version 4.0.3 
> library(tidyverse)
-- Attaching packages -------------------------------------------------------------------------------- tidyverse 1.3.0 --
v ggplot2 3.3.3.9000     v purrr   0.3.4     
v tibble  3.0.5          v dplyr   1.0.3     
v tidyr   1.1.2          v stringr 1.4.0     
v readr   1.4.0          v forcats 0.5.0     
-- Conflicts ----------------------------------------------------------------------------------- tidyverse_conflicts() --
x tidyr::extract()   masks magrittr::extract()
x dplyr::filter()    masks stats::filter()
x dplyr::lag()       masks stats::lag()
x purrr::set_names() masks magrittr::set_names()
Warning messages:
1: package ‘tidyverse’ was built under R version 4.0.3 
2: package ‘tidyr’ was built under R version 4.0.3 
3: package ‘readr’ was built under R version 4.0.3 
4: package ‘purrr’ was built under R version 4.0.3 
5: package ‘stringr’ was built under R version 4.0.3 
6: package ‘forcats’ was built under R version 4.0.3 
> R.version
               _                           
platform       x86_64-w64-mingw32          
arch           x86_64                      
os             mingw32                     
system         x86_64, mingw32             
status                                     
major          4                           
minor          0.2                         
year           2020                        
month          06                          
day            22                          
svn rev        78730                       
language       R                           
version.string R version 4.0.2 (2020-06-22)
nickname       Taking Off Again            
> sessionInfo()
R version 4.0.2 (2020-06-22)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19041)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252    LC_MONETARY=English_Australia.1252
[4] LC_NUMERIC=C                       LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] forcats_0.5.0      stringr_1.4.0      dplyr_1.0.3        purrr_0.3.4        readr_1.4.0        tidyr_1.1.2       
 [7] tibble_3.0.5       ggplot2_3.3.3.9000 tidyverse_1.3.0    magrittr_2.0.1     phyloseq_1.34.0    breakaway_4.7.3   

loaded via a namespace (and not attached):
 [1] nlme_3.1-151        fs_1.5.0            matrixStats_0.57.0  lubridate_1.7.9.2   RColorBrewer_1.1-2 
 [6] progress_1.2.2      httr_1.4.2          tools_4.0.2         backports_1.2.1     R6_2.5.0           
[11] vegan_2.5-7         DBI_1.1.1           BiocGenerics_0.36.0 mgcv_1.8-33         colorspace_2.0-0   
[16] permute_0.9-5       rhdf5filters_1.2.0  ade4_1.7-16         withr_2.4.0         tidyselect_1.1.0   
[21] gridExtra_2.3       prettyunits_1.1.1   compiler_4.0.2      cli_2.2.0           rvest_0.3.6        
[26] Biobase_2.50.0      xml2_1.3.2          scales_1.1.1        minqa_1.2.4         XVector_0.30.0     
[31] pkgconfig_2.0.3     lme4_1.1-26         dbplyr_2.0.0        rlang_0.4.10        readxl_1.3.1       
[36] rstudioapi_0.13     generics_0.1.0      jsonlite_1.7.2      biomformat_1.7.0    Matrix_1.2-18      
[41] fansi_0.4.2         Rcpp_1.0.6          munsell_0.5.0       S4Vectors_0.28.1    Rhdf5lib_1.12.0    
[46] ape_5.4-1           lifecycle_0.2.0     stringi_1.5.3       MASS_7.3-53         zlibbioc_1.36.0    
[51] rhdf5_2.34.0        plyr_1.8.6          grid_4.0.2          parallel_4.0.2      ggrepel_0.9.1      
[56] crayon_1.3.4        lattice_0.20-41     Biostrings_2.58.0   haven_2.3.1         splines_4.0.2      
[61] multtest_2.46.0     hms_1.0.0           pillar_1.4.7        igraph_1.2.6        boot_1.3-25        
[66] corpcor_1.6.9       reshape2_1.4.4      codetools_0.2-18    mixOmics_6.14.0     stats4_4.0.2       
[71] reprex_0.3.0        glue_1.4.2          data.table_1.13.6   modelr_0.1.8        vctrs_0.3.6        
[76] nloptr_1.2.2.2      foreach_1.5.1       cellranger_1.1.0    gtable_0.3.0        assertthat_0.2.1   
[81] xfun_0.20           broom_0.7.3         RSpectra_0.16-0     survival_3.2-7      rARPACK_0.11-0     
[86] iterators_1.0.13    tinytex_0.29        IRanges_2.24.1      ellipse_0.4.2       cluster_2.1.0      
[91] statmod_1.4.35      ellipsis_0.3.1

**Comment by EmilyHoedt** [comment]
I fixed the issue. I replaced R and R studio with the latest version (R v4.0.3). I then had in issue with tidyr not being recognized in the architecture. Although it was uninstalled the folder was still present with the R/win-library so the reinstallation of tidyr wasn't working. The folder couldn't be deleted, so I renamed it and from there tidyr and DivNet installation all worked smoothly.

I wanted to share the information here in case anyone else struggles in future.

Thank you!
